client/src/components/VideoForm.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import validUrl from "valid-url";
+
+function VideoForm({ videoData, setVideoData, setFetchData }) {
+  const [errorMessage, setErrorMessage] = useState("");
+
+  function addVideoHandler(event) {
+    event.preventDefault();
+    function validateUrl(urlObject) {
+      return (
+        validUrl.isUri(urlObject) &&
+        (urlObject.startsWith("https://www.youtube.com") ||
+          urlObject.startsWith("https://youtu.be") ||
+          urlObject.startsWith("https://m.youtube.com") ||
+          urlObject.startsWith("https://youtube.com/"))
+      );
+    }
+
+    const formTitle = event.target.form.title.value;
+    const formUrl = event.target.form.url.value;
+
+    const cleanedUrl = formUrl.includes("&") ? formUrl.split("&")[0] : formUrl;
```

Comment:
What's this code cleaning up? What does this code achieve? Maybe worth a comment, with an example of what's being removed, just above the code :)

Reply:
Thank you for reminding me to put that comment! I have added the comment with an example. I had to remove the end bits so that I get to display the thumbnails of the YouTube videos in my app.